projects/2019_12_16_MultiSig_QT_Wallet.md

```diff
@@ -0,0 +1,90 @@
+
+## 1. Project Relevance to the PKT Network
+> _"The [objective of The Network Steward](https://pkt-cash.github.io/www.pkt.cash/steward/) is to reduce barriers of entry to network / VPN providers in the pursuit of increased individual autonomy and privacy"_
+
+The creation of a new multi-sig QT PKT wallet can achieve the following benefits for the PKT Network:
+
+* Expand the adoption of the PKT Network to users who do not have the technical skills, time, or resources to operate the current [open-source files of a wallet](http://github.com/pkt-cash/pktwallet), and run a [full node](https://github.com/pkt-cash/pktd).
+* Higher security & control, backup feature, and joint-account capability, provided by the multi-sig-transaction accessibility.
+* Increase the density of the PKT Network nodes, to contribute to building the infrastructure of a functional decentralized network.
+
+In the list of examples of [Projects Supported by The Network Steward](http://github.com/pkt-cash/ns-projects/blob/master/readme.md), the first example is a wallet. As of the date of preparation of this proposal, the only available wallet is a [text-based wallet](http://github.com/pkt-cash/pktwallet).
+
+## 2. Project Deliverables
+The project will create new software for a standalone application, which should be downloaded and installed from an Installer. The installation will come bundled and ready to run a full-node, [PKTD](https://github.com/pkt-cash/pktd), as well as a [PKT Wallet](http://github.com/pkt-cash/pktwallet) instance.
+
+This deliverables are:
+* New software for a standalone application.
+* The software will be available under the [ISC](https://opensource.org/licenses/ISC) license.
+* Any artistic non-software assets will be available under the [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/) license.
+* The maintainer of this software will be: Vishnu Seesahai
+* The software will be hosted on https://github.com/pkt-cash, if the network steward wishes it so.
+
+### 2.1 Development overview
+The development will be carried out by using the QT Software toolkit due to its history of reliability, portability, and low memory usage.
+
+The wallet will have a graphical user interface (GUI).
+Compatible operating systems: MacOS, Unix.
+The software will have future interoperability with Lightning Network.
+
+For the avoidance of doubt, the development will not be an SPV wallet, and it will not have features for colored coins. This software will not have a low-resource footprint because it will run a full node.
+
+### 2.2 Wallet target features
+1. Send and receive transactions.
+2. Send transactions to multiple addresses at once.
+3. Batch transactions for large UTXO input sets.
+4. Sign transactions.
+5. Verify transactions.
+6. Generate new regular addresses and new multi-sig addresses.
+7. Export private keys and import new private keys.
+8. Encrypt and decrypt messages.
```

Comment:
Can you clarify what are `messages` in this context? I also see a  ✉️message tab on the wireframe.

Reply:
This feature is similar to electrum wallet, where you can paste in strings, ie. messages, and encrypt / decrypt them with your key pair. Not super useful for this version of the wallet. But, the groundwork is in place to execute encrypted chats from your wallet, similar to [BCH](https://news.bitcoin.com/newly-launched-keyport-platform-enables-bch-powered-encrypted-messaging/)